hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/DeploymentManagement.java

```diff
@@ -516,4 +519,54 @@ static DeploymentRequestBuilder deploymentRequest(final String controllerId, fin
     @PreAuthorize(SpringEvalExpressions.HAS_AUTH_READ_TARGET)
     boolean hasPendingCancellations(@NotEmpty String controllerId);
 
+    /**
+     * Invalidates a given {@link DistributionSet}. The invalidation always
+     * cancels all auto assignments referring this {@link DistributionSet} and
+     * can not be undone. Optionally, all rollouts and actions referring this
+     * {@link DistributionSet} can be canceled.
+     *
+     * @param distributionSetInvalidation
+     *            defines the {@link DistributionSet} and options what should be
+     *            canceled
+     */
+    @PreAuthorize(SpringEvalExpressions.HAS_AUTH_UPDATE_REPOSITORY)
```

Comment:
Additionally please check if we need to use "SpringEvalExpressions.HAS_AUTH_READ_REPOSITORY_AND_UPDATE_REPOSITORY" auth rule in case there is no READ Repository rights present in current authentication, please verify Mgmt API